bin/gql-gen.ts

```diff
@@ -40,53 +40,54 @@ async function main(): Promise<void> {
         const cwd = process.cwd();
         const lib = libDir(cwd);
 
+        // patch up codegen template to create optional properties
+        const codegenTemplate = path.join(cwd, "node_modules", "graphql-codegen-typescript-client", "dist", "index.js");
+        if (await fs.pathExists(codegenTemplate)) {
+            let contents = (await fs.readFile(codegenTemplate)).toString();
```

Comment:
Should all this be in a try/catch block?

Reply:
Not sure. I’d like it to fail if this fails.